src/Appwrite/Platform/Modules/Proxy/Http/Rules/Function/Create.php

```diff
@@ -192,7 +192,7 @@ public function action(string $domain, string $functionId, string $branch, Respo
             $queueForCertificates
                 ->setDomain(new Document([
                     'domain' => $rule->getAttribute('domain'),
-                    'domainType' => 'function',
+                    'domainType' => $rule->getAttribute('deploymentResourceType', $rule->getAttribute('type')),
```

Comment:
For all existing rules, `type` and  `deploymentResourceType` are empty. Dont think we can use the domainType from those attributes